tests/repositoryArgs.py

```diff
@@ -50,51 +52,59 @@ class DefaultMapper(unittest.TestCase):
     inputs with different mappers then the butler will not assume a default mapper and _getDefaultMapper
     will return None."""
 
+    def setUp(self):
+        self.testDir = os.path.join(ROOT, 'DefaultMapper')
+        self.tearDown()
+        os.makedirs(self.testDir)
+
+    def tearDown(self):
+        if os.path.exists(self.testDir):
+            shutil.rmtree(self.testDir)
+
+    def _setup(self, mapper1, mapper2):
+        repo1Root = os.path.join(self.testDir, 'repo1')
+        repo2Root = os.path.join(self.testDir, 'repo2')
+        butler = dp.Butler(outputs=(dp.RepositoryArgs(root=repo1Root, mapper=mapper1)))
+        del butler
```

Comment:
sometimes it clarifies intention (a new one is created later), and other times it gets the linter to stop complaining about unused variables. and sometimes both.